app/javascript/controllers/profile_controller.js

```diff
@@ -1,6 +1,12 @@
 import { Controller } from "@hotwired/stimulus";
 
 export default class extends Controller {
+  removeActive() {
+    const profileSections = document.querySelector(".profile-sections");
+    const active = profileSections.querySelector(".active");
+    active.classList.remove("active");
+  }
+  
   connect() {
     let video = document.querySelector(".video");
     let post = document.querySelector(".post");
```

Comment:
Explain what u changed please.
Where is active class & what it does?

Reply:
it is to show the section selected as active
![image](https://github.com/BrightCoders-Institute/s10a1-proyecto-final-team-03-s10a1-proyecto-final/assets/112733483/66f20fc4-5bfa-413c-b847-f65162f22978)
here the item on the right is selected, before that i set, it just maintain the color brown on click in the mobile view on the browser, after another click the brow is lose and it is not diferenciable what section it is active